### PR TITLE
ban user upload illegal document

### PR DESCRIPTION
**Pull request by vannam1996**

### CODE REVIEW

**Comment by vannam1996** [comment]
image

**Comment by vannam1996** [comment]
image

File: config/settings.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ user:
   default_buy: 1
 documents:
   file: ".pdf"
-  illegal: 5
+  illegal: 1
```
Comment (lamphan95) [inline config/settings.yml]: sửa lại 5

**Comment by vannam1996** [comment]
ready

**Pull request merged**